.husky/pre-commit

```diff
@@ -1,4 +1 @@
-#!/usr/bin/env sh
```

Comment:
After manually testing a git commit to trigger the pre-commit hook, I saw the following message:

<img width="1185" alt="Screenshot 2025-06-02 at 9 59 31 PM" src="https://github.com/user-attachments/assets/570ddef4-66db-48d5-b488-37c9169b2de2" />

Hence I removed this from the script

Reply:
More justification for this: https://github.com/typicode/husky/releases/tag/v9.0.1